app/controllers/users_controller.rb

```diff
@@ -1,15 +1,28 @@
 class UsersController < ApplicationController
+
+  before_action :logged_in_user, only: [:index, :edit, :update]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :admin_user, only: :destroy
+  before_action :find_user, only: [:show, :edit, :update, :destroy]
+
+  ORDER_BY_ATTRIBUTE = "created_at"
+
+  def index
+    @users = Kaminari.paginate_array(User.all.order ORDER_BY_ATTRIBUTE).page(params[:page])
```

Comment:
==>Như này cho ngắn gọn em he
```ruby
@users = User.order(ORDER_BY_ATTRIBUTE).page(params[:page])
  .per Settings.items_per_pag
```

Reply:
Đại ca ơi e lên mạng tra phải dùng `paginate_array` để chuyển Active Record sau khi được order thành dạng có thể `paginate` được ạ, nếu không sẽ dính lỗi `undefined method per for #<User::ActiveRecord_Relation:0x00007fbbf9ba27c0>`

Reply:
do đệ đang dùng cả 2 gem để phân trang đóa @@